lib/slim/parser.rb

```diff
@@ -533,5 +545,13 @@ def expect_next_line
       next_line || syntax_error!('Unexpected end of file')
       @line.strip!
     end
+
+    def process_embedded_attrs(attr_string)
+      attrs = [:html, :attrs]
+      @line = attr_string
+      parse_attributes(attrs)
+      @line = @orig_line
+      attrs
+    end
   end
```

Comment:
I don't like how this parsing is done. I would prefer if the parse_attributes method is somehow generalized to work also for the embedded attributes.